examples/framework_usage/main.rs

```diff
@@ -158,12 +151,6 @@ async fn main() {
             })
         })
         .options(options)
-        .client_settings(|b| {
-            b.intents(
-                serenity::GatewayIntents::non_privileged()
-                    | serenity::GatewayIntents::MESSAGE_CONTENT,
-            )
-        })
```

Comment:
And this

Reply:
Branch conflicted somehow it has been restored in the latest commit, same goes for the listener option.